### PR TITLE
Onboard to pre-release extensions

### PR DESCRIPTION
**Pull request by dbanck**
Running the `./build/preview` script will now only update the version (and LS version if supplied):

```diff
-  "version": "2.26.0",
+  "version": "2.26.2023050315",
```

We supply the `--pre-release` to `vsce package` and `vsce publish` as outlined [here](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prerelease-extensions).

### CODE REVIEW

**Review by jpogran — APPROVED** [review]

File: build/preview [hunk 1]
```diff
@@ -7,26 +7,24 @@ ROOT_RELATIVE_DIR=$(dirname "${SCRIPT_RELATIVE_DIR}")
...
-
-# set preview info in package.json
-(cat package.json | jq --arg VER $VERSION --arg LANGVER $LANGUAGE_SERVER_VERSION --arg NAME "$DISPLAY_NAME" --arg DESC "$DESCRIPTION" '
+# Update versions in package.json
+(cat package.json | jq --arg VER $VER --arg LANGVER $LANGUAGE_SERVER_VERSION '
 .version=$VER |
-.langServer.version=$LANGVER |
-.preview=true |
-.name="terraform-preview" |
-.icon="assets/icons/terraform_logo_mark_dark_universal.png" |
```
Comment (radeksimko) [inline build/preview:26]: Shall we also remove this icon now?

**Review by radeksimko — APPROVED** [review]
LGTM, my only nit-picky note which is less related to the diff/PR is that shell scripts like `build/preview` typically have an extension like `*.sh` or `*.bash`.

File: build/preview (hunk 1 above)
Comment (dbanck) [reply build/preview:26]: Good catch, did some cleanup

**Pull request merged**